app/assets/stylesheets/alchemy/jquery.datetimepicker.scss

```diff
@@ -0,0 +1,545 @@
+$datepicker_current_bg: $light-blue !default;
+$datepicker_current_color: $text-color !default;
+$datepicker_current_box_shadow: $blue 0 1px 3px 0 inset !default;
+$datepicker_hover_bg: $light_yellow !default;
+$datepicker_hover_color: $text-color !default;
+$datepicker_today_bg: $very-light-blue !default;
+$datepicker_today_color: $text-color !default;
+$datepicker_header_color: $text-color !default;
+$datepicker_day_bg: $light-gray !default;
+$datepicker_day_border: 1px solid $medium-gray !default;
+$datepicker_day_color: $text-color !default;
+
+.xdsoft_datetimepicker {
+  box-shadow: 0 5px 15px -5px rgba(0, 0, 0, 0.506);
+  background: #fff;
+  border-bottom: 1px solid #bbb;
+  border-left: 1px solid #ccc;
+  border-right: 1px solid #ccc;
+  border-top: 1px solid #ccc;
+  border-radius: $default-border-radius;
+  color: $text-color;
+  padding: 8px;
+  padding-left: 0;
+  padding-top: 2px;
+  position: absolute;
+  z-index: 9999;
+  -moz-box-sizing: border-box;
```

Comment:
You could actually leave that out. `box-sizing` should be understood by every browser these days, especially firefox.